### ISSUE TITLE
Move to a CG?

### ISSUE THREAD

**Issue by jyasskin** [explainers-by-googlers/storage-access-for-fedcm#4]
Given @bvandersloot-mozilla's co-authorship of the Lightweight FedCM proposal, which seems to depend on this, I suspect there's enough support to move this proposal to a CG.

**Comment by johannhof** [comment]
Ah, my bad for linking here from the TAG review response, we should probably deprecate this repo. We did get this adopted in Privacy CG but @martinthomson gave us guidance to merge it as an explainer into the main Storage Access repo instead of transferring the repo, IIRC. So I moved it here: https://github.com/privacycg/storage-access/blob/main/explainers/storage-access-for-fedcm.md

It does sound like a separate repo might be nice but yeah, I tried to follow chairs guidance for now. Happy to improve it.

**Comment by jyasskin** [comment]
Yeah, no need for a separate repo. Can you update your TAG review to point at the PrivacyCG version of this? (I'd do it, but I might miss things that changed between the two versions.)
